Sources/StreamVideoSwiftUI/CallViewModel.swift

```diff
@@ -328,6 +328,8 @@ open class CallViewModel: ObservableObject {
     ///  - maxParticipants: An optional integer representing the maximum number of participants allowed in the call.
     ///  - startsAt: An optional date when the call starts.
     ///  - backstage: An optional request for setting up backstage.
+    ///  - hasVideo: A boolean indicating if the call will be video or only audio. Still requires appropriate
```

Comment:
This is not enough - we should also put this in the `IncomingCall` model, so customers can provide a different UI based on it. This should be added in `CallEventsHandler.checkForCallEvents`.

Reply:
Good point!